src/main/java/dukeclasses/Duke.java

```diff
@@ -0,0 +1,42 @@
+package dukeclasses;
+
+/**
+ * Contains a product name dukeClasses.Duke, a personal assistant chat bot that keeps track of various tasks that needs,
+ * to be done. It classifies tasks into deadline, todo and event.
+ * It can add, delete, list all the tasks, find task with a keyword and mark them as done.
+ */
+public class Duke {
+
+    /**
+     * The main method to start Duke.
+     *
+     * @param args args to pass in.
+     */
+
+    public static void main(String[] args) {
+
+        FastReader fr = new FastReader();//Handles input by the user
```

Comment:
I don't think you need to add this comment here as it does not add value to the code.

Reply:
Alright thank you! I'll take note of it